components/mailbox/src/init.spec.js

```diff
@@ -1338,3 +1338,45 @@ describe("Mailbox Integration: Show draft message in Email thread", () => {
       .should("be.visible");
   });
 });
+
+describe("Display Participants in thread row", () => {
+  beforeEach(() => {
+    cy.intercept(
+      "GET",
+      "https://web-components.nylas.com/middleware/manifest",
+      {
+        fixture: "mailbox/manifest.json",
+      },
+    );
+    cy.intercept("GET", "https://web-components.nylas.com/middleware/account", {
+      fixture: "mailbox/account.json",
+    });
+    cy.intercept("GET", "https://web-components.nylas.com/middleware/labels", {
+      fixture: "mailbox/labels.json",
+    });
+
+    cy.intercept(
+      "GET",
+      "https://web-components.nylas.com/middleware/threads?view=expanded&not_in=trash&limit=13&offset=0&in=inbox",
+      {
+        fixture: "mailbox/threads/threadWithDraft.json",
+      },
+    );
```

Comment:
You could use a batchIntercept here to reduce the amount of code.
```suggestion
    cy.batchIntercept("GET", {
      "**/middleware/manifest": "mailbox/manifest",
      "**/middleware/account": "mailbox/account",
      "**/middleware/labels": "mailbox/labels",
      "**/middleware/threads*": "mailbox/threads/threadWithDraft",
    });
```